evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -44,27 +46,35 @@ class UnknownShard(Exception):
     pass
 
 
+@to_dict
+def deserialize_header_bytes(header_bytes):
```

Comment:
This function feels like something that should be a `classmethod` on the `CollationHeader` class.

Reply:
Thank you for this great suggestion! I will change it.